### DISCUSSION TITLE
"No server is currently available" or NGHTTP2_REFUSED_STREAM

### DISCUSSION THREAD

**Discussion by Matt-17** [community/community#112238] [Copilot Conversations]
### Select Topic Area

Bug

### Body

It's starting to get annoying. 

Lately I often get error messages like this: 

 <p><strong>No server is currently available to service your request.</strong></p> (inside a html document)

or 

NGHTTP2_REFUSED_STREAM

If I wait a while it's working sometimes.

**Comment by ThinkerDreamer** [comment]
I just had this problem, and I saw I was running an older version of the Copilot extension in VSCode, so I updated it (to `1.172.0`) and after a reload of VSCode, it seems fixed.

**Reply by Matt-17** [reply]
Thank you. I'm using Visual Studio (without Code) and my current version is 1.156.0. An update is not yet available. 
I will look into this. Thank you so far.

**Reply by pablisch** [reply]
I am having the same problems and I am already using version 1.172.0

**Reply by pablisch** [reply]
I noticed the latest release date is today and switched to the pre-release version and for now at least, it is working.

**Comment by YannHulot** [comment]
I am getting the same error. It's happening intermittently. It works fine for 5 minutes and then stops working for 10 minutes. Then back to working again.

I am vscode (latest version), and using the GH copilot extension also on the latest version.

**Comment by jessica-brooks** [comment]
I am getting the same issue with the Co-pilot extension in PyCharm (JetBrains). I'm definitely using the latest version and have re-auth'd my device but that hasn't fixed it.
